modApiServer/test/org/aion/api/server/ApiAionTest.java

```diff
@@ -421,4 +428,85 @@ public void testHashRate() {
         assertTrue(api.setReportedHashrate(Double.toString(hashRate), ""));
         assertEquals(impl.getBlockMiner().getHashrate() + hashRate, Double.parseDouble(api.getHashrate()), 0.001);
     }
+
+    @Test
+    public void testEthSignTransaction() {
+        Address addr = new Address(Keystore.create("testPwd"));
+
+        AccountManager.inst().unlockAccount(addr, "testPwd", 50000);
+
+        Address toAddr = new Address(Keystore.create("testPwd"));
+
+        JSONObject tx = new JSONObject();
+        tx.put("from", addr.toString());
+        tx.put("gasPrice", "20000000000");
+        tx.put("gas", "21000");
+        tx.put("to", toAddr.toString());
+        tx.put("value", "500000");
+        tx.put("data", "");
+
+        JSONArray jsonArray = new JSONArray();
+        jsonArray.put(tx);
+        jsonArray.put(addr);
+
+        RpcMsg rpcMsg = web3Api.eth_signTransaction(jsonArray);
+        assertNotNull(rpcMsg);
+
+        JSONObject result = (JSONObject)rpcMsg.getResult();
+        assertNotNull(result);
```

Comment:
I think we could be a bit more stringent with this test.  I suggest adding one or both of these:

1. Verify the sub-fields within `result.get("tx")` against the original transaction that the test signed to make sure they're correct in the signed transaction
2. Verify that the signed transaction is valid by calling eth_sendRawTransaction with`result ` and seeing that it doesn't return null / throw an exception